<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="icon_text_widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>251</width>
      <height>841</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 167);
background-color: rgb(59, 59, 59);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>55</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>19</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: white;
}
</string>
         </property>
         <property name="text">
          <string>      UEL LIBRARY</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Dashboard</string>
         </property>
         <property name="icon">
          <iconset resource="../../LIBRARY/resources.qrc">
           <normaloff>:/icons/icons/grid.svg</normaloff>:/icons/icons/grid.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Books</string>
         </property>
         <property name="icon">
          <iconset resource="../../LIBRARY/resources.qrc">
           <normaloff>:/icons/icons/book-open.svg</normaloff>:/icons/icons/book-open.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Readers</string>
         </property>
         <property name="icon">
          <iconset resource="../../LIBRARY/resources.qrc">
           <normaloff>:/icons/icons/eye.svg</normaloff>:/icons/icons/eye.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Transfer</string>
         </property>
         <property name="icon">
          <iconset resource="../../LIBRARY/resources.qrc">
           <normaloff>:/icons/icons/users.svg</normaloff>:/icons/icons/users.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Community</string>
         </property>
         <property name="icon">
          <iconset resource="../../LIBRARY/resources.qrc">
           <normaloff>:/icons/icons/globe.svg</normaloff>:/icons/icons/globe.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../LIBRARY/resources.qrc">
           <normaloff>:/icons/icons/settings.svg</normaloff>:/icons/icons/settings.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>SIGN OUT</string>
         </property>
         <property name="icon">
          <iconset resource="../../LIBRARY/resources.qrc">
           <normaloff>:/icons/icons/log-out.svg</normaloff>:/icons/icons/log-out.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>971</width>
      <height>841</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QWidget" name="header_widget" native="true">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>344</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../LIBRARY/resources.qrc">
             <normaloff>:/icons/icons/align-justify.svg</normaloff>:/icons/icons/align-justify.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>29</width>
             <height>35</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(135, 135, 135);</string>
           </property>
           <property name="text">
            <string>Welcome to your page.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Hello, Vy !</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>0</y>
          <width>321</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="styleSheet">
            <string notr="true">QlineEdit{
  padding-left:20px;
  border:1px-solid gray;
  border-radius: 20px;
}</string>
           </property>
           <property name="placeholderText">
            <string>Search here....</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(135, 135, 135);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Admin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(135, 135, 135);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../LIBRARY/resources.qrc">:/icons/icons/user.svg</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>100</y>
          <width>941</width>
          <height>741</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>941</width>
          <height>741</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="maximumSize">
          <size>
           <width>841</width>
           <height>741</height>
          </size>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>240</y>
            <width>161</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>dashboard</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <property name="maximumSize">
          <size>
           <width>841</width>
           <height>741</height>
          </size>
         </property>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>250</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>books</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="maximumSize">
          <size>
           <width>841</width>
           <height>741</height>
          </size>
         </property>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>280</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>readers</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <property name="maximumSize">
          <size>
           <width>841</width>
           <height>741</height>
          </size>
         </property>
         <widget class="QTableWidget" name="transferTable">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>250</y>
            <width>911</width>
            <height>481</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QHeaderView::section{
    font-weight: bold;
    background-color: black;
    color: white;
}

QTableWidget{
    alternate-background-color: #B0EDFB;
    background-color: #F4F9FA;
}
</string>
          </property>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>student_id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>book_id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>borrow_date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>due_date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>status</string>
           </property>
          </column>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>134</x>
            <y>75</y>
            <width>91</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../LIBRARY/resources.qrc">:/icons/icons/alert-circle.svg</pixmap>
          </property>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>70</y>
            <width>221</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white;
background-color: rgb(187, 191, 255);
border-radius: 15px;
border: 1px solid #dddddd;
</string>
          </property>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>55</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../LIBRARY/resources.qrc">:/icons/icons/clock.svg</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>30</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Borrowing : 3</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>70</y>
            <width>221</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white;
background-color: rgb(255, 135, 161);
border-radius: 15px;
border: 1px solid #dddddd;
</string>
          </property>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>55</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../LIBRARY/resources.qrc">:/icons/icons/alert-circle.svg</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>40</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Overdue : 3</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>70</y>
            <width>221</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white;
background-color: rgb(225, 253, 255);
border-radius: 15px;
border: 1px solid #dddddd;
</string>
          </property>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>55</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../LIBRARY/resources.qrc">:/icons/icons/thumbs-up.svg</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>40</y>
             <width>171</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Already returned : 4</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <property name="maximumSize">
          <size>
           <width>841</width>
           <height>741</height>
          </size>
         </property>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>280</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>community</string>
          </property>
         </widget>
        </widget>
       </widget>
       <zorder>stackedWidget</zorder>
       <zorder>layoutWidget</zorder>
       <zorder>layoutWidget</zorder>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>icon_text_widget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../LIBRARY/resources.qrc"/>
 </resources>
 <connections/>
</ui>
